templates/iqe-trigger.yml

```diff
@@ -25,7 +29,16 @@ objects:
                 cpu: 200m
                 memory: 256Mi
 
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: image-builder
+
 parameters:
   - name: WEBHOOK_URL
     value: ''
     required: true
+  - name: JOB_SUFFIX
+    description: "Random string used as job suffix"
+    generate: expression
+    from : "[a-zA-Z0-9]{12}"
```

Comment:
This should not be needed, the job should be replaced. Generate seems like it would add an ever increasing amount of jobs.

Reply:
Ah misunderstanding on my part, this is a one-off job run, we don't have to do this for the maintenance job because that's a cronjob which confused me.